e2e/tests/plugins/displayLayout/dispayLayout.e2e.spec.js

```diff
@@ -0,0 +1,137 @@
+    });
+    test('alpha-numeric widget telemetry value exactly matches latest telemetry value received in fixed time', async ({ page, openmctConfig }) => {
+        await page.goto('./', { waitUntil: 'networkidle' });
+        const { myItemsFolderName } = openmctConfig;
+
+        await page.locator('button:has-text("Fixed Timespan")').click();
+        await page.locator('[data-testid="conductor-modeOption-realtime"]').click();
+
+        await createDomainObjectWithDefaults(page, {
+            type: 'Sine Wave Generator',
+            name: "Test Sine Wave Generator"
+        });
+        const pageURL = page.url();
+        const sineWaveGeneratorIdentifier = pageURL.split('/').pop().split('?')[0];
+
+        await page.locator(`text=${myItemsFolderName}`).first().click();
+
+        await createDomainObjectWithDefaults(page, {
+            type: 'Display Layout',
+            name: "Test Display Layout"
+        });
+        await page.locator('[title="Edit"]').click();
+
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').click();
+        await page.dragAndDrop('text=Test Sine Wave Generator', '.l-layout__grid-holder');
+
+        await page.locator('button[title="Save"]').click();
+        await page.locator('text=Save and Finish Editing').click();
+
+        const getTelemValuePromise = new Promise(resolve => page.exposeFunction('getTelemValue', resolve));
+
+        await subscribeToTelemetry(page, sineWaveGeneratorIdentifier);
+        await changeToFixedTime(page);
+
+        const subscribeTelemValue = await getTelemValuePromise;
+        const displayLayoutValuePromise = await page.waitForSelector(`text="${subscribeTelemValue}"`);
+        const displayLayoutValue = await displayLayoutValuePromise.textContent();
+        const trimmedDisplayValue = displayLayoutValue.trim();
+
+        await expect(trimmedDisplayValue).toBe(subscribeTelemValue);
+    });
+});
+
+/**
+ * Util for subscribing to a telemetry object by object identifier
+ * @param {import('@playwright/test').Page} page
+ * @param {string} objectIdentifier identifier for object
+ */
+async function subscribeToTelemetry(page, objectIdentifier) {
+    await page.evaluate(async (telemetryIdentifier) => {
+        const telemetryObject = await window.openmct.objects.get(telemetryIdentifier);
+        const metadata = window.openmct.telemetry.getMetadata(telemetryObject);
+        const formats = await window.openmct.telemetry.getFormatMap(metadata);
+        window.openmct.telemetry.subscribe(telemetryObject, (obj) => {
+            const sinVal = obj.sin;
+            const formattedSinVal = formats.sin.format(sinVal);
+            window.getTelemValue(formattedSinVal);
+        });
+    }, objectIdentifier);
+}
+
+/**
+ * This function uses the time conductor to change to a fixed time of 1 minute
+ * @param {import('@playwright/test').Page} page
+ */
+async function changeToFixedTime(page) {
```

Comment:
Check out these helper functions: https://github.com/nasa/openmct/blob/master/e2e/tests/functional/plugins/timeConductor/timeConductor.e2e.spec.js

@unlikelyzero Good candidates for appActions additions?